src/helpers.js

```diff
@@ -30,9 +30,9 @@ export function dummyVmsFilter(vms, uiVms) {
 
 export function toReadableNumber(number) {
     if (number < 1) {
-        return number.toFixed(2);
+        return Math.floor(number * 100) / 100;
     } else {
-        const fixed1 = number.toFixed(1);
+        const fixed1 = Math.floor((number * 10) / 10);
         return (number - fixed1 === 0) ? number.toFixed(0) : fixed1;
```

Comment:
I believe this has to be also adjusted?
```suggestion
        return (number - fixed1 === 0) ? Math.floor(number) : fixed1;
```